expo/gbe/templates/gbe/landing_page.tmpl

```diff
@@ -18,6 +18,7 @@
                     {% if profile and tickets %}
                     <div class='landing_box'>
 			 {% include 'gbe/incl_tickets.tmpl' %}
+       {% include 'gbe/profile/incl_performer.tmpl' %}
```

Comment:
Sounds good will look into it.
On Aug 7, 2016 12:34, Jon Kiparsky <notifications@github.com> wrote:In expo/gbe/templates/gbe/landing_page.tmpl:
> @@ -18,6 +18,7 @@
>                      {% if profile and tickets %}
>                      <div class='landing_box'>
>  			 {% include 'gbe/incl_tickets.tmpl' %}
> +       {% include 'gbe/profile/incl_performer.tmpl' %}

It looks like the template indentation is messed up here - not your code, so not a blocker but this is a good opportunity to do some cleanup if you want. (1) consistent 4-space indents throughout (2) eliminate tabs, use spaces only. 

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.